src/app/app.component.html

```diff
@@ -1,8 +1,9 @@
 <div class="content">
   <div class="content-inside">
     <app-header></app-header>
-    <app-breadcrumbs></app-breadcrumbs>
-    <app-courses-page></app-courses-page>
+    <app-breadcrumbs *ngIf="authenticationService.isAuthenticated"></app-breadcrumbs>
```

Comment:
The best way is creating of method in component (*.component.ts) that will call service method. In this example you can create getter in component. Check all views in project where you call methods of service from view and replace to methods of components.

Reply:
done